code/src/index.css

```diff
@@ -1,13 +1,27 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  margin: 0 auto;
+  background-image: url('./background.jpg');
+  background-repeat: no-repeat;
+  background-size: 100%;
+  font-family: fantasy;
+  color: #f53e9a;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1{
+  font-size: 30px;
+}
+
+.survey {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+  margin-top: 15%;
+  margin-right: 10%;
+  margin-left: 50%;
+}
```

Comment:
Nice to see you managed to create such a distinctive style with only 27 lines of CSS, well done! Very versatile 😄

Reply:
Yes, that part of my project is not finished, I just have to start the next week's project - but as soon as possible I am going to fix it!